### ISSUE TITLE
how can you connect to existing on-disk database?

### ISSUE THREAD

**Issue by zachcp** [juji-io/datalevin#17]
@huahaiy 

What is the best way to initiate a connection to a saved, on-disk database?

```clojure

;; example from readme:  initiate DB/add data/close conn
;;
(require '[datalevin.core :as d])

(def conn (d/create-conn {} "/tmp/datalevin-test"))

(d/transact! conn
             [{:name "Frege", :db/id -1, :nation "France", :aka ["foo" "fred"]}
              {:name "Peirce", :db/id -2, :nation "france"}
              {:name "De Morgan", :db/id -3, :nation "English"}])

(d/close! conn)

;; how do I re-establish a DB connection to the existing DB?
;; reopen?
(d/open-db-conn "/tmp/datalevin-test")
```

**Comment by huahaiy** [comment]
I would suggest to manage a Datalevin connection just like any other stateful connections, e.g. connection to Postgres database, etc.  For instance, you could use any of the Clojure state management libraries, such as  component, mount, integrant, and so on. The idea is to keep an open connection around. 

One initiates a connection to a saved, on-disk database the same way as one initiates a new database. `(create-conn schema dir)`. If you are not updating the schema, just use `nil` for it. `(create-conn nil dir)`. 

P.S. using a nil seems strange. Let me change the function to `(create-conn dir)` instead.

**Comment by huahaiy** [comment]
Released version 0.3.1 to correct the argument list for `create-conn`, also update doc string so it is clear that existing DB should be opened with the same function. Closing.
